packages/tools/nodeEditor/src/components/preview/previewManager.ts

```diff
@@ -283,13 +288,32 @@ export class PreviewManager {
             dir1.parent = this._lightParent;
         }
     }
+    private _hdrSkyBox: Mesh;
+    private _hdrTexture: HDRCubeTexture;
+    private _prepareBackgroundHDR() {
+        if (this._hdrTexture == null) {
+            this._hdrTexture = new HDRCubeTexture("room.hdr", this._scene, 512);

```

Comment:
Please use `room.env` instead, it will save the conversion from .hdr at loading time.